src/runtime/link-accounts-flow.js

```diff
@@ -286,3 +286,62 @@ export class LinkSaveFlow {
         });
   }
 }
+
+export class LoginPromptFlow {
+  /**
+   * @param {!./deps.DepsDef} deps
+   */
+  constructor(deps) {
+    /** @private @const {!./deps.DepsDef} */
+    this.deps_ = deps;
+
+    /** @private @const {!Window} */
+    this.win_ = deps.win();
+
+    /** @private @const {!web-activities/activity-ports.ActivityPorts} */
+    this.activityPorts_ = deps.activities();
+
+    /** @private @const {!../components/dialog-manager.DialogManager} */
+    this.dialogManager_ = deps.dialogManager();
+
+    /** @private {?Promise} */
+    this.openViewPromise_ = null;
+
+    /** @private @const {!ActivityIframeView} */
+    this.activityIframeView_ = new ActivityIframeView(
+        this.win_,
+        this.activityPorts_,
+        feUrl('/loginpromptiframe'),
+        feArgs({
+          publicationId: deps.pageConfig().getPublicationId(),
+          productId: deps.pageConfig().getProductId(),
```

Comment:
Possibly this is coming later if this is just a boilerplate. But heads up: we will need to pass `entitlements` value here. `deferred-account-flow.js` is probably a better model for this feature as a boilerplate. Feel free to keep and change it later, of course.

Reply:
done